src/pipx/venv.py

```diff
@@ -301,7 +301,11 @@ def install_package_no_deps(self, package_or_url: str, pip_args: List[str]) -> s
             f"determining package name from {package_or_url!r}", self.do_animation
         ):
             old_package_set = self.list_installed_packages()
-            cmd = ["install"] + ["--no-dependencies"] + pip_args + [package_or_url]
+            cmd = (
+                ["--no-input", "install", "--no-dependencies"]
+                + pip_args
+                + [package_or_url]
+            )
```

Comment:
```suggestion
            cmd = [
                "--no-input",
                "install",
                "--no-dependencies",
                *pip_args,
                package_or_url,
            ]
```

This is growing too complicated

Reply:
I left some in the form of `cmd = [<snip>] + cmd` alone instead of changing it to `cmd = [<snip>, *cmd]`.